app/assets/javascripts/pages/errors.coffee

```diff
@@ -171,3 +185,48 @@ $('form#notify').submit ->
       $('.messageTextarea').val('')
       return
   false
+
+PubSub.subscribe('assigned.website', (ev, website)->
+  $.getJSON Routes.api_v1_website_path(website.id), (data) ->
+    $('#current-website').render data
+)
+
+$('.tab').hide()
+$('.tab2, .tab3').addClass('disabled')
+
+$('#back').on 'click', () ->
+  $('.tab').hide()
+  manipulateWizard(2)
+
+$('#finish').on 'click', () ->
+  location.href = '/errors'
+
+$('#platform a').on 'click', (e) ->
+  $('.tab').hide()
+  $('#'+this.name).show()
+  manipulateWizard(3)
+
+manipulateWizard = (n) ->
+  if n != 0
+    $('.stepwizard-row a').removeClass('btn-primary')
+    $('.stepwizard-row a').addClass('btn-default')
+    $('.stepwizard a[href="#step-' + n + '"]').tab 'show'
+    $('.stepwizard-row a[href="#step-' + n + '"]').removeClass 'btn-default'
+    $('.stepwizard-row a[href="#step-' + n + '"]').addClass 'btn-primary'
+  return
+
+$('#modalWebsite').submit (e) ->
+  e.preventDefault()
+  form = $('#modalWebsite')
+  $.ajax
+    url: Routes.api_v1_websites_url()
+    type: 'post'
+    dataType: 'json'
+    data: { website: { domain: form.find('#domain').val(), title: form.find('#title').val() } }
```

Comment:
avand in vedere ca e acelasi request ai putea sa faci o functie si sa dai parametru functia de success. Uite aici: http://stackoverflow.com/a/13286241/759476